lib/evss/letters/service.rb

```diff
@@ -1,9 +1,22 @@
 # frozen_string_literal: true
+require 'evss/base_service'
+
 module EVSS
   module Letters
-    class Service
-      # TODO(AJD): only mock letter service for now...
-      def initialize(_headers)
+    class Service < EVSS::BaseService
+      BASE_URL = "#{Settings.evss.url}/wss-lettergenerator-services-web/rest/letters/v1"
+
+      def get_letters
+        raw_response = get ''
+        EVSS::Letters::LettersResponse.new(raw_response)
+      end
+
+      def get_letter_for_user_by_type(type)
+        get type
+      end
+
+      def self.breakers_service
+        BaseService.create_breakers_service(name: 'EVSS/Letters', url: BASE_URL)
```

Comment:
@aub is breakers concerned with high-level services or does it report on individual endpoints as well? This may also depend on if the one evss url is a reverse proxy for a set of services (claims, letters, gi bill status etc.)